test/snapshots/no-useless-interpolation.mjs.md

```diff
@@ -0,0 +1,207 @@
+# Snapshot report for `test/no-useless-interpolation.mjs`
+
+The actual snapshot is saved in `no-useless-interpolation.mjs.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## Invalid #1
+      1 | `${"foo"}`
+
+> Error 1/1
+
+    `␊
+    > 1 | \`${"foo"}\`␊
+        |    ^^^^^ Unnecessary interpolation of "foo" in template string.␊
```

Comment:
can someone help me fix the report position? @fisker I saw in [your branch](https://github.com/fisker/eslint-plugin-unicorn/commits/no-useless-interpolation) you had also a problem with it

Reply:
I understood, there is no problem is snapshots with report location

![image](https://user-images.githubusercontent.com/7361780/147133763-ff21ea78-3157-4444-b59a-8f306c3f6d9f.png)

This happens because there are backslashes before ` symbols in snapshots, can you install something like this for jest https://github.com/ikatyang/jest-snapshot-serializer-raw but for ava to fix this?

Reply:
There is nothing to fix, it's correct result, just one offset.